reconcile/prometheus_rules_tester.py

```diff
@@ -77,6 +78,7 @@ def get_prometheus_rules(cluster_name):
         ):
             continue
 
+        ob.aggregate_shared_resources(n, "openshiftResources")
```

Comment:
nit: `"openshiftResources" is repeated enough that it could probably be a constant.